sorc/syndat_maksynrc.fd/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+list(APPEND fortran_src
+  maksynrc.f
+)
+
+if(CMAKE_Fortran_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -assume noold_ldout_format")
+endif()
+
+set(exe_name syndat_maksynrc.x)
+add_executable(${exe_name} ${fortran_src})
+target_link_libraries(
+  ${exe_name}
+  w3nco::w3nco_4)
```

Comment:
True.  But there is no call to any bacio library function.  I think it might have been an error.
The build would have failed if the program did require it.